op-e2e/actions/l2_runtime.go

```diff
@@ -42,21 +42,23 @@ type Runtime struct {
 	outputOnL1               bindings.TypesCheckpointOutput
 	txHash                   common.Hash
 	receipt                  *types.Receipt
+	l1BlockDelta             uint64
 }
 
-func defaultRuntime(gt *testing.T) Runtime {
+type SetupSequencerTestFunc = func(t Testing, sd *e2eutils.SetupData, log log.Logger) (*L1Miner, *L2Engine, *L2Sequencer)
+
+func defaultRuntime(gt *testing.T, setupSequencerTest SetupSequencerTestFunc) Runtime {
 	t := NewDefaultTesting(gt)
 	dp := e2eutils.MakeDeployParams(t, defaultRollupTestParams)
```

Comment:
`defaultRollupTestParams` is declared in test file, it causes compile error. Let's move the declaration to a file that is not test file. Although it causes diff from OP, we should adjust it when significant problem occurs like compile error.